index.js

```diff
@@ -149,6 +149,12 @@ const preloadResources = opt => {
     if (/^data:|blob:/i.test(responseUrl)) return;
     const ct = response.headers()["content-type"] || "";
     const route = responseUrl.replace(basePath, "");
+    if (cacheAjaxRequests && ct.includes("json")) {
+      if (uniqueResources.has(responseUrl)) return;
+      const json = await response.json();
+      ajaxCache[route] = json;
+      uniqueResources.add(responseUrl);
```

Comment:
Also we do not need to `preconnect` to third-party server if it is already cached.

Reply:
The number of options leads to high number of possible cases, that is why I do not like to add options 😞. Zero-config is the best